package.json

```diff
@@ -122,7 +122,7 @@
     "sinon": "^12.0.1",
     "sinon-chai": "^3.7.0",
     "source-map-support": "^0.5.20",
-    "sqlite3": "^5.0.2",
+    "sqlite3": "npm:@vscode/sqlite3@^5.0.7",
```

Comment:
Why not just use vscode/sqlite3 directly? It's a devDependency anyway, users will have to switch on their end too.